Tests/test_SeqIO_QualityIO.py

```diff
@@ -176,46 +194,58 @@ def check_qual_char(self, filename, good_count, count):
 
 # Now add methods at run time... these FASTQ files will be rejected
 # by both the low level parser AND the high level SeqRecord parser:
-tests = [("diff_ids", 2),
-         ("no_qual", 0),
-         ("long_qual", 3),
-         ("short_qual", 2),
-         ("double_seq", 3),
-         ("double_qual", 2),
-         ("tabs", 0),
-         ("spaces", 0),
-         ("trunc_in_title", 4),
-         ("trunc_in_seq", 4),
-         ("trunc_in_plus", 4),
-         ("trunc_in_qual", 4),
-         ("trunc_at_seq", 4),
-         ("trunc_at_plus", 4),
-         ("trunc_at_qual", 4)]
+tests = [
+    ("diff_ids", 2),
+    ("no_qual", 0),
+    ("long_qual", 3),
+    ("short_qual", 2),
+    ("double_seq", 3),
+    ("double_qual", 2),
+    ("tabs", 0),
+    ("spaces", 0),
+    ("trunc_in_title", 4),
+    ("trunc_in_seq", 4),
+    ("trunc_in_plus", 4),
+    ("trunc_in_qual", 4),
+    ("trunc_at_seq", 4),
+    ("trunc_at_plus", 4),
+    ("trunc_at_qual", 4),
+]
 for base_name, good_count in tests:
+
     def funct(name, c):
         f = lambda x: x.check_all_fail("Quality/error_%s.fastq" % name, c)  # noqa: E731
         f.__doc__ = "Reject FASTQ with %s" % name.replace("_", " ")
         return f
-    setattr(TestFastqErrors, "test_%s" % (base_name),
-            funct(base_name, good_count))
+
+    setattr(TestFastqErrors, "test_%s" % (base_name), funct(base_name, good_count))
     del funct
 
 # Now add methods for FASTQ files which will be rejected by the high
 # level SeqRecord parser, but will be accepted by the low level parser:
-tests = [("del", 3, 5),
-         ("space", 3, 5),
-         ("vtab", 0, 5),
-         ("escape", 4, 5),
-         ("unit_sep", 2, 5),
-         ("tab", 4, 5),
-         ("null", 0, 5)]
+tests = [
+    ("del", 3, 5),
+    ("space", 3, 5),
+    ("vtab", 0, 5),
+    ("escape", 4, 5),
+    ("unit_sep", 2, 5),
+    ("tab", 4, 5),
+    ("null", 0, 5),
+]
 for base_name, good_count, full_count in tests:
+
     def funct(name, c1, c2):
-        f = lambda x: x.check_qual_char("Quality/error_qual_%s.fastq" % name, c1, c2)  # noqa: E731
+        f = lambda x: x.check_qual_char(
+            "Quality/error_qual_%s.fastq" % name, c1, c2
+        )  # noqa: E731
```

Comment:
Flake8 doesn't like the placement of the ``# noqa: E731`` pragma:

```
$ flake8
./Tests/test_SeqIO_QualityIO.py:238:9: E731 do not assign a lambda expression, use a def
```

I would guess it needs to go on the line with the ``lambda`` itself?

Reply:
Yes, I missed that one out, sorry, I have relocated it. Thank You.